source/content/guides/secrets/03-terminus-plugin.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Pantheon Secrets Guide
+subtitle: Terminus Secrets Manager Plugin
+description: Terminus plugin for Pantheon Secrets
+contributors: [stovak]
+contenttype: [guide]
+innav: [true]
+categories: [secrets]
+cms: [drupal, wordpress]
+audience: [development]
+product: [secrets]
+integration: [--]
+tags: [reference, cli, local, terminus, workflow]
+permalink: docs/guides/secrets/terminus-plugin
+reviewed: "2024-05-01"
+---
+
+**IMPORTANT: when Pantheon Secrets goes into GA, it will be part of terminus core. Until then, this plugin information covers [THIS REPOSITORY]( https://github.com/pantheon-systems/terminus-secrets-manager-plugin).**
+
+# Terminus Secrets Manager Plugin
+
+### To Install:
+
+```
+terminus self:plugin:install terminus-secrets-manager-plugin
+```
+
+### To Uninstall:
+
+```
+terminus self:plugin:uninstall terminus-secrets-manager-plugin
+```
+
+### To Update:
+
+```
+terminus self:plugin:update terminus-secrets-manager-plugin
+```
+
```

Comment:
Are we going to include instructions on how to use the Secrets Manager plugin commands here?

Reply:
I added a link to the command reference.